logcat/api/logcat.api

```diff
@@ -12,33 +13,34 @@ public final class logcat/AndroidLogcatLogger$Companion {
 	public static synthetic fun installOnDebuggableApp$default (Llogcat/AndroidLogcatLogger$Companion;Landroid/app/Application;Llogcat/LogPriority;ILjava/lang/Object;)V
 }
 
-public final class logcat/AndroidLogcatLoggerKt {
-}
-
 public final class logcat/LogPriority : java/lang/Enum {
 	public static final field ASSERT Llogcat/LogPriority;
 	public static final field DEBUG Llogcat/LogPriority;
 	public static final field ERROR Llogcat/LogPriority;
 	public static final field INFO Llogcat/LogPriority;
 	public static final field VERBOSE Llogcat/LogPriority;
 	public static final field WARN Llogcat/LogPriority;
+	public static fun getEntries ()Lkotlin/enums/EnumEntries;
 	public final fun getPriorityInt ()I
 	public static fun valueOf (Ljava/lang/String;)Llogcat/LogPriority;
 	public static fun values ()[Llogcat/LogPriority;
 }
 
-public final class logcat/LogcatKt {
-	public static final fun logcat (Ljava/lang/Object;Llogcat/LogPriority;Ljava/lang/String;Lkotlin/jvm/functions/Function0;)V
-	public static final fun logcat (Ljava/lang/String;Llogcat/LogPriority;Lkotlin/jvm/functions/Function0;)V
-	public static synthetic fun logcat$default (Ljava/lang/Object;Llogcat/LogPriority;Ljava/lang/String;Lkotlin/jvm/functions/Function0;ILjava/lang/Object;)V
-	public static synthetic fun logcat$default (Ljava/lang/String;Llogcat/LogPriority;Lkotlin/jvm/functions/Function0;ILjava/lang/Object;)V
+public final class logcat/Logcat {
+	public static final fun logcat (Ljava/lang/Object;Llogcat/LogPriority;Ljava/lang/String;Llogcat/MessageSupplier;)V
+	public static final fun logcat (Ljava/lang/String;Llogcat/LogPriority;Llogcat/MessageSupplier;)V
+	public static synthetic fun logcat$default (Ljava/lang/Object;Llogcat/LogPriority;Ljava/lang/String;Llogcat/MessageSupplier;ILjava/lang/Object;)V
+	public static synthetic fun logcat$default (Ljava/lang/String;Llogcat/LogPriority;Llogcat/MessageSupplier;ILjava/lang/Object;)V
 	public static final fun outerClassSimpleNameInternalOnlyDoNotUseKThxBye (Ljava/lang/Object;)Ljava/lang/String;
 }
 
 public abstract interface class logcat/LogcatLogger {
 	public static final field Companion Llogcat/LogcatLogger$Companion;
+	public static fun install (Llogcat/LogcatLogger;)V
```

Comment:
Added `@JvmStatic` a few places so you don't have to dig through the `.Companion` object in java to get to these methods.